docs/workers.md

```diff
@@ -269,6 +269,13 @@ permission. It is done by adding this to the manifest:
 }
 ```
 
+## trash-files worker
+
+This worker is used only by the stack: when the user asks to clean the trash,
+the stack will delete the files from CouchDB and put a job for this worker. The
+deletion of files in Swift is slow, and can be done via this worker
+asynchronously.
+
```

Comment:
Maybe you can explain here the differences between afero/swiftv1 (no worker) and swift v2/v3 (with worker) for later reference